src/ClientModel/SamplingDataPostProcessor.cpp

```diff
@@ -56,7 +56,8 @@ class SamplingDataPostProcessor {
   // Filled by ProcessSamples.
   absl::flat_hash_map<ThreadID, ThreadSampleData> thread_id_to_sample_data_;
   absl::flat_hash_map<uint64_t, CallstackInfo> id_to_resolved_callstack_;
-  absl::flat_hash_map<std::vector<uint64_t>, uint64_t> resolved_callstack_to_id_;
+  absl::flat_hash_map<std::pair<std::vector<uint64_t>, CallstackInfo::CallstackType>, uint64_t>
```

Comment:
Nit: A custom struct instead of a pair would be more expressive here.

Reply:
Done as part of the heterogeneous-lookup change.